src/bttrx_wifi.cpp

```diff
@@ -152,9 +152,14 @@ void BTTRX_WIFI::setup(BTTRX_CONTROL *control) {
   }
   bttrx_control_ = control;
 
+  Serial.println("");
   Serial.println("Configuring access point...");
 
-  WiFi.softAP(WIFI_SSID, WIFI_PASSWORD);
+  string ssid = BTTRX_WIFI::buildSSID(WIFI_SSID, bttrx_control_->getCallsign());
```

Comment:
Please change name of `WIFI_SSID` to `WIFI_SSID_PREFIX`

Reply:
Fixed with https://github.com/bt-trx/firmware/pull/40/commits/5927cc77ca56e754ee0d6930ec983344373e2f63